wasm/bls-utils/src/runtime/fvm_injection.rs

```diff
@@ -0,0 +1,28 @@
+use fvm_wasm_instrument::gas_metering;
+use parity_wasm::elements::{serialize_to_file, Module};
+use std::{fs::read, path::Path};
+
+
+// const STACK_LIMIT: u32 = 128;
+
+pub fn inject_fvm_modules() {
```

Comment:
Can we make this function take in a generic &Path object -- rather than harcoding the path ?